### DISCUSSION TITLE
Filter boolean category

### DISCUSSION THREAD

**Discussion by theMackabu** [pocketbase/pocketbase#239] [Q&A] [answered]
Is it possible to filter with a boolean category? Ive tried ?filter=(type='true'), ?filter=(type=1), ?filter=(type=true) and none of them seemed to work.

**Comment by ganigeorgiev** [comment, marked as answer]
What do you mean by _"boolean category"_?

If `type` from your example is a boolean field, then `type = true` should work.

The only thing that may cause an issue in your case is if you are not properly url encoding the filter query parameter (the js-sdk do that for you), or in other words - the url encoded version of the filter would be `?filter=type%3Dtrue`.

**Reply by theMackabu** [reply]
ah, I seem to have been encoding it wrong
